.gitignore

```diff
@@ -27,6 +28,7 @@ mkmf.log
 */tmp/
 */vendor/
 */pkg/
+yard_docs
```

Comment:
How about using `/docs/` instead of `yard_docs`?
We generate our docs only by YARD.

Reply:
There are [files in a `doc` directory](https://github.com/ruby-gnome/ruby-gnome/tree/master/doc), for Heroku installation with rsvg2 and Release instructions. For the `yard_docs` pathname, it would not be ambiguous with the `doc` directory. 

In [`.github/workflows/doc.yml`](https://github.com/ruby-gnome/ruby-gnome/blob/master/.github/workflows/doc.yml), a  `yard_docs` dir is used in the `bundle exec yard doc` step, before renaming the directory to a location for publishing the documentation. This directory would have a different structure than the `yard_docs` dir that would be generated with the Rakefile in the patch. It seemed like a common pathname that was distinct to the pathname for the project documentation

Reply:
The `yard_doc` name wasn't considered carefully.

Anyway, I'll merge this as-is. We can organize directory struct as follow-up tasks.

Reply:
fwiw for the idea of combining the documentation for Gir definitions and their implementation in RG with the main Yard docs for RG: I think Yard process the source code, when generating docs? 

I may have been able to figure out some aspects for type mapping when ruby-gnome is already loaded, e.g for classes that provide an `INVOKERS` constant and/or `INITIALIZE_INVOKERS` and for modules that provide an `INVOKERS` constant under the module's singleton class. If  a Yard plugin would be able to load the Ruby-GNOME API for which it's generating documentation, it could process those values to generate documentation for the method mapped to each Invoker instance?

To iterate on these values, the documentation plugin could need to be provided with a module name, where it would begin iteration, ideally after Yard has processed the source definitions.

This itself would not provide complete API coverage? For instance, definitions provided e.g any dynamically loaded definitions under `GLib` might not be reached if the plugin would only be processing for Invoker instances?

If the plugin was provided with a module name or a list of module names, or if the module names could be computed from what Yard has processed, perhaps there may be a way to run the API walker after Yard has already processed the source code. For each class or module definition beginning at each of the named modules, if  Yard has not found documentation for that class, module, and/or any public (or other) method on the same, the API walker may be able to add documentation from any definitions located via Ruby GNOME?

If I can figure out how the plugin might iterate on any dynamically loaded definitions (??) under GLib, hopefully then it could provide complete API coverage, with tests. Then, perhaps there would only be a question of how to produce something that Yard could use for its YRI structures then adding that to Yard's parse, in a transformation on each definition?

Reply:
How about creating a new issue  https://github.com/ruby-gnome/ruby-gnome/issues/new or new discussion https://github.com/ruby-gnome/ruby-gnome/discussions/new for further discussion instead of reusing a closed pull request?